packages/backend/src/db/announcement/announcement.schema.ts

```diff
@@ -0,0 +1,28 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type AnnouncementDocument = Announcement & Document;
+
+@Schema({ timestamps: true })
+export class Announcement {
+  @Prop()
+  title: string;
+
+  @Prop()
+  description: string;
+
+  @Prop()
+  author: string;
```

Comment:
Might be good to have this as an `ObjectId`, so that we can refer to the `User` in case they ever change their name.

Reply:
Good idea, Ill change that!